keyboards/acheron/doddle60/keymaps/via/keymap.c

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2020 Alvaro "Gondolindrim" Volpato <alvaro.augusto.volpato@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+   [0] = LAYOUT_iso(
```

Comment:
You'd need to do this on zvecr's suggestion, too.

```suggestion
   [0] = LAYOUT_60_iso_split_bs_rshift(
```

Reply:
Done in 1f035aa07